docs/index.bs

```diff
@@ -3113,7 +3113,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
           1. Otherwise, set |callback| to the result of [=convert to an ECMAScript value|converting to an ECMAScript value=] |eventListenerCallback|.
           1. If [$IsCallable$](|callback|) is false, then return false.
 
-              Note: [=Event listener/Callback=] objects that use {{handleEvent}} are assumed to be non-empty. This avoids calling the {{handleEvent}} getters, which could modify the event listeners during this check.
+              Note: [=event listener/Callback=] objects that use <code>handleEvent</code> are assumed to be non-empty. This avoids calling the <code>handleEvent</code> getters, which could modify the event listeners during this check.
```

Comment:
I think these could be `{{EventListener/handlEvent(event)}}` and keep the auto-linking

Reply:
done.